containers/podman-desktop
# fix: added text beside icon for create pods

### What does this PR do?
Adds the text "Create Pod" beside the icon in the Containers List screen.

### Screenshot/screencast of this PR


![Screenshot from 2023-11-29 21-59-32](https://github.com/containers/podman-desktop/assets/106850342/9c44af8b-3c92-4b56-9bb9-2b88abd7631a)


### What issues does this PR fix or reference?
Issue #4413



### How to test this PR?
UI test only.  Go to the Containers tab and select a container.  When the buttons appear near the top of the screen the words "Create Pod" will display on the button next to the pods icon.
